wetmap/src/components/newModals/setting/view.tsx

```diff
@@ -0,0 +1,28 @@
+import screenData from '../screenData.json';
+import React from 'react';
+import styles from './style.module.scss';
+import ButtonIcon from '../../reusables/buttonIcon';
+import Icon from '../../../icons/Icon';
+
+type SettingsProps = {
+  onClose:      () => void
+};
+
+export default function SettingsView(props: SettingsProps) {
+  return (
+    <>
+      <div className={styles.buttonBack}>
+        <ButtonIcon
+          icon={<Icon name="chevron-left" />}
+          className="btn-lg text-gray"
+          onClick={(e) => {
```

Comment:
why all the extra check here? 
 onClick={props.onClose} 
seems to work just fine on its own?

Reply:
I'm still seeing these being imported, why did you mark this as resolved with out making changes or at least documenting your reasoning in here as a response?